data/tokens.json

```diff
@@ -211,109 +211,197 @@
       }
     },
     "typography": {
-      "H1": {
+      "family": {
+        "base": {
+          "value": "Mulish",
+          "type": "fontFamily"
+        }
+      },
+      "size": {
+        "h1": {
+          "value": "40",
+          "type": "fontSize"
+        },
+        "h2": {
+          "value": "32",
+          "type": "fontSize"
+        },
+        "h3": {
+          "value": "24",
+          "type": "fontSize"
+        },
+        "h4": {
+          "value": "20",
+          "type": "fontSize"
+        },
+        "body": {
+          "value": "16",
+          "type": "fontSize"
+        },
+        "caption": {
+          "value": "14",
+          "type": "fontSize"
+        },
+        "footnote": {
+          "value": "12",
+          "type": "fontSize"
+        }
+      },
+      "weight": {
+        "heading": {
+          "value": "700",
+          "type": "fontWeight"
+        },
+        "base": {
+          "value": "500",
+          "type": "fontWeight"
+        }
+      },
+      "letterSpacing": {
+        "base": {
+          "value": "0",
+          "type": "letterSpacing"
+        }
+      },
+      "lineHeight": {
+        "xlarge": {
+          "value": "40",
+          "type": "lineHeight"
+        },
+        "large": {
+          "value": "32",
+          "type": "lineHeight"
+        },
+        "medium": {
+          "value": "24",
+          "type": "lineHeight"
+        },
+        "small": {
+          "value": "16",
+          "type": "lineHeight"
+        }
+      },
+      "paragraphSpacing": {
+        "base": {
+          "value": "0",
+          "type": "paragraphSpacing"
+        }
+      },
+      "textDecoration": {
+        "base": {
+          "value": "none",
+          "type": "textDecoration"
+        }
+      },
+      "textCase": {
+        "base": {
+          "value": "none",
+          "type": "textCase"
+        }
+      },
+      "h1": {
         "value": {
-          "fontFamily": "Mulish",
-          "fontWeight": "700",
-          "lineHeight": "40",
-          "fontSize": "40",
-          "letterSpacing": "0%",
-          "paragraphSpacing": "0",
-          "textDecoration": "none",
-          "textCase": "none"
+          "fontFamily": "{typography.family.base}",
```

Comment:
Previous format: `{global.typography.family.base}`

Adding `global` in `typography.family.base` namespace is unnecessary since `global` gets stripped from the JSON object in the `transformed-tokens.json`